src/app/shared/model/distribution.model.ts

```diff
@@ -88,29 +100,21 @@ export class DistributionModel extends BaseModel {
     return this.parent.create('prx:distributions', {}, data);
   }
 
+  // swap new episode with this.url
   saveRelated(): Observable<boolean[]> {
-    if (this.podcast && this.podcast.isNew) {
+    if (this.podcast && this.podcast.isNew && this.url) {
       let oldModel = this.podcast;
-
-      // CMS should actually have created the podcast in feeder
-      return this.loadExternal().flatMap(() => {
-        oldModel.copyTo(this.podcast);
+      this.loadRelated('podcast', true).subscribe((newModel: FeederPodcastModel) => {
```

Comment:
Yeah, seems that CMS isn't returning the story_distribution right away, after it's automagically created along with the story (there's now an open issue for that).

I'm currently refactoring parts of the BaseModel to allow for these records that are automagically created (the FeederPodcast, StoryDistribution, and FeederEpisode are all created on the backend).  Basically, after the parent model finishes saving and goes to `saveRelated()`, it will check for related that need to be reloaded first.

Getting close(r)!